mods.json

```diff
@@ -17,13 +17,14 @@
             "version" : "1.0.1"
         },
         "hardcoded-config-injector" : {
-            "name" : "Hardcoded Config Injector",
-            "description" : "Allows you to change hardcoded objects inside a json file.",
-            "page" : [{ "name" : "GitHub", "url" : "https://github.com/CCDirectLink/hardcoded-config-injector" }],
-            "archive_link" : "https://github.com/CCDirectLink/hardcoded-config-injector/archive/master.zip",
-            "hash" : { "sha256" : "674186444840cda641d9720b681ba254777b2625f4333b320454ad116be0b9b3" },
-            "version" : "0.5.0"
-        },
+		"name": "hardcoded-config-injector",
+		"description": "Inject into hardcoded CrossCode code values.",
+		"license": null,
+		"page": [{ "name" : "GitHub", "url" : "https://github.com/CCDirectLink/hardcoded-config-injector/"}],
+		"archive_link": "https://github.com/CCDirectLink/hardcoded-config-injector/archive/v0.1.0.zip",
+		"hash": { "sha256" : "5c85fb567cf381ceb03548eb49fc074634f5f425836422525926f46610dc1c9f" },
```

Comment:
sha sum does not match `8c4b50f9719c85498a784cba5131119a4719ea3249971f3c1ed9dec501a854be`